client/src/utils/mutations.js

```diff
@@ -19,6 +19,54 @@ export const LOGIN_USER = gql`
   }
 `;
 
+//adding bio
```

Comment:
I need to see how this code will be used on the front end first so some of it might change